Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs

```diff
@@ -1357,8 +1357,9 @@ public new Padding Padding
         /// <summary>
         /// Gets and sets the text associated with the control.
         /// </summary>
-        public override string? Text
+        public new virtual string Text
         {
+            // Replaced override with "new virtual" since inheritance keeps producing null warnings which ever way
             get => _comboBox.Text;
```

Comment:
This can be null in the newer frameworks...

Reply:
This hooks up with my earlier explanation on 
`[Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupComboBox.cs](https://github.com/Krypton-Suite/Standard-Toolkit/pull/1459/files/1c2450229996836dbc697259fc8b7034a0d5cf4d#diff-cd2e7f5863063dd2fd805078ad0f5c60ac4a82b081311edef75278f303ec47cd)`